app/cdap/components/ParsingDrawer/Components/ParsingHeaderActionTemplate/index.tsx

```diff
@@ -14,56 +14,56 @@
  * the License.
  */
 
-import { Box } from '@material-ui/core';
-import { useStyles } from 'components/ParsingDrawer/styles';
-import React, { ChangeEvent } from 'react';
+import { Box, Typography } from '@material-ui/core';
 import { parseImportedSchemas } from 'components/AbstractWidget/SchemaEditor/SchemaHelpers';
+import { useStyles } from 'components/ParsingDrawer/styles';
 import T from 'i18n-react';
+import React, { ChangeEvent } from 'react';
+import { importIcon } from './importicon';
+import { IParsingHeaderActionTemplateProps } from './types';
 
-export default function(props) {
+export default function({
+  handleSchemaUpload,
+  setErrorOnTransformation,
+}: IParsingHeaderActionTemplateProps) {
   const classes = useStyles();
-
   const handleFile = (event: ChangeEvent<HTMLInputElement>) => {
     const schemaFile = event.target.files[0];
     const reader = new FileReader();
-
     reader.readAsText(schemaFile, 'UTF-8');
-
     reader.onload = (evt) => {
       try {
         const fileContents = JSON.parse(evt.target.result.toString());
         const importedSchemas = parseImportedSchemas(fileContents);
         const schema = importedSchemas[0] && importedSchemas[0].schema;
-        props.handleSchemaUpload(schema);
+        handleSchemaUpload(schema);
       } catch (e) {
-        props.setErrorOnTransformation({
+        setErrorOnTransformation({
           open: true,
-          message: T.translate('features.WranglerNewParsingDrawer.importSchemaErrorMessage'),
+          message: T.translate(
+            'features.WranglerNewUI.WranglerNewParsingDrawer.importSchemaErrorMessage'
+          ),
         });
       }
     };
   };
-
   return (
-    <Box>
+    <Box className={classes.importSchemaIconText}>
       <input
+        data-testId="fileinput"
         id="file"
         type="file"
         accept=".json"
         style={{ display: 'none' }}
         onChange={handleFile}
         className={classes.pointerStyles}
       />
-      <label htmlFor="file" className={classes.pointerStyles}>
-        <img
-          className={classes.importIconStyles}
-          src="/cdap_assets/img/import.svg"
-          alt="import schema icon"
-        />
-        <span className={classes.importSchemaTextStyles}>
-          {T.translate('features.WranglerNewParsingDrawer.importSchema')}
-        </span>
+      <label htmlFor="file" className={classes.importSchemaLabel}>
```

Comment:
update this htmlFor based on the renaming of the id on the file input

Reply:
assign it to unit tester